### PR TITLE
B t35312 adjust styling for support card

### PR DESCRIPTION
**Pull request by sakutademos**
**Ticket:** https://yaits.demos-deutschland.de/T35312

**Description:** This PR adjusts styling for the support card as well as fixes the issue where all cards were displayed instead of only the visible ones.

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/components/support/DpSupport.vue [hunk 1]
```diff
@@ -24,12 +24,12 @@ All rights reserved
     </h3>
     <ul
       class="u-mb-0_75"
-      :class="contactCount === 1 ? '' : 'grid lg:grid-cols-3 gap-3'">
+      :class="{ 'grid lg:grid-cols-3 gap-3': visibleContacts.length !== 1 }">
```
Comment (gruenbergerdemos) [inline client/js/components/support/DpSupport.vue:27]: How does this work? The classes are only added if the condition is true?

File: client/js/components/support/DpSupportCard.vue [hunk 2]
```diff
@@ -26,8 +26,7 @@ All rights reserved
       v-if="email"
       v-text="email" />
     <div
-      v-if="reachability.service"
-      class="u-pb-0_75">
+      v-if="reachability.service">
```
Comment (gruenbergerdemos) [inline client/js/components/support/DpSupportCard.vue:29]: v-if should be ordered first

**Review by gruenbergerdemos — CHANGES_REQUESTED** [review]

File: client/js/components/support/DpSupport.vue (hunk 1 above)
Comment (sakutademos) [reply client/js/components/support/DpSupport.vue:27]: yes, right

File: client/js/components/support/DpSupportCard.vue (hunk 2 above)
Comment (sakutademos) [reply client/js/components/support/DpSupportCard.vue:29]: the class attribute is not needed any more, it was removed

Comment (gruenbergerdemos) [reply client/js/components/support/DpSupportCard.vue:29]: Oh, right, my mistake

**Review by gruenbergerdemos — APPROVED** [review]
lgtm

**Pull request merged**